recipes/eups/deactivate.sh

```diff
@@ -0,0 +1,25 @@
+# unsetup any products to keep env clean
```

Comment:
I'd appreciate credit for my existing work here: https://github.com/beckermr/stackvana-core/blob/master/recipe/stackvana_deactivate.sh